include/main.h

```diff
@@ -1,64 +1,58 @@
+#ifndef HANGMAN_MAIN_H
+#define HANGMAN_MAIN_H
+
 ///////////////////////////
 // VARIABLE DECLARATIONS //
 ///////////////////////////
 
 // hangman banner
-const char *hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
+char const* const hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
+
+char const* const FRESH = "________\n |/   |\n |\n |\n |\n |\n_|_\n";
+char const* const FIRST = "________\n |/   |\n |   (_)\n |\n |\n |\n_|_\n";
+char const* const SECOND = "________\n |/   |\n |   (_)\n |    |\n |    |\n |\n_|_\n";
+char const* const THIRD = "________\n |/   |\n |   (_)\n |   /|\n |    |\n |\n_|_\n";
+char const* const FOURTH =  "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |\n_|_\n";
+char const* const FIFTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   /\n_|_\n";
+char const* const SIXTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   / \\\n_|_\n";
 
-// hangman drawing based on how many lives lost
-// using macros because initiating an array of pointers is not a constant expression, which can't be initialized as a global variable
-#define FRESH "________\n |/   |\n |\n |\n |\n |\n_|_\n"
-#define FIRST "________\n |/   |\n |   (_)\n |\n |\n |\n_|_\n"
-#define SECOND "________\n |/   |\n |   (_)\n |    |\n |    |\n |\n_|_\n"
-#define THIRD "________\n |/   |\n |   (_)\n |   /|\n |    |\n |\n_|_\n"
-#define FOURTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |\n_|_\n"
-#define FIFTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   /\n_|_\n"
-#define SIXTH "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   / \\\n_|_\n"
+char const* const stages[7] = {FRESH, FIRST, SECOND, THIRD, FOURTH, FIFTH, SIXTH};
 
 // dynamic variables
-char *word; // holds the random word chosen at the beginning of the game
-char guess[2]; // holds the 1 letter guess
+char const* word; // holds the random word chosen at the beginning of the game
```

Comment:
`const` quite frankly should be default, and a variable should only be denoted mutable (no `const`) if it *must* be changed.